RecoEgamma/EgammaElectronProducers/python/gsfElectronSequence_cff.py

```diff
@@ -8,9 +8,13 @@
 
 from RecoEgamma.EgammaElectronProducers.gsfElectronModules_cff import *
 gsfElectronSequence = cms.Sequence(ecalDrivenGsfElectronCores*ecalDrivenGsfElectrons*gsfElectronCores*gsfElectrons)
```

Comment:
As I commented in the previous iteration,
I expect that all files that are touched will have ```cms.Sequence(modules ``` -> ```cms.Sequence(someTask) ``` conversion.

Reply:
OK. I will complete the others.